### PR TITLE
REPL tool for debugging

### PR DESCRIPTION
**Pull request by kcasas**
/cc @zendesk/mintegrations

### Description
* Created a wrapper for psysh. Run the command `bin/console <subdomain> <email> <api token>`. This would create an instance of `Zendesk\Api\HttpClient` on $client variable.

### References
* psysh: http://psysh.org

### Risks
* [low] tool only used api token for authentication. Support for other authentication should follow.

### CODE REVIEW

**Comment by miogalang** [comment]
@kcasas Please fix the commits. Squash them maybe? I see a commit "urg"

File: bin/ConsoleCommand.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Zendesk;
+
+require 'DocMethodMatcher.php';
```
Comment (samgavinio) [inline bin/ConsoleCommand.php]: Any way to add these to the load path automatically?

File: bin/Shell.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+use Psy\TabCompletion\Matcher\ObjectMethodsMatcher;
+
+class Shell extends \Psy\Shell
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function getTabCompletionMatchers()
+    {
+        $tabCompletionMatchers = parent::getTabCompletionMatchers();
```
Comment (samgavinio) [inline bin/Shell.php]: Please document here a bit why there is a need to filter out ObjectMethodsMatcher and ObjectAttributesMatcher.

File: bin/ConsoleCommand.php (hunk 1 above)
Comment (samgavinio) [reply bin/ConsoleCommand.php]: Can we move these source files within  the`src/Zendesk/API` namespace and just keep `console` in the bin directory?

**Comment by samgavinio** [comment]
Super cool addition to the client. Some annotations in the matcher would be nice since it's sort of hard to follow. And do we need tests for this?
